### ISSUE TITLE
MoreAll command

### ISSUE THREAD

**Issue by monodemono** [ProgVal/Limnoria#1060]
Just had an idea... What about a MoreAll command? "(2 more messages)" MoreAll would just do both messages, one right after the other, without requiring another More to be executed. There can be a configuration for the maximum number of messages MoreAll will do, to keep the bot from flooding if someone does it even by accident when there's a lot more messages. Also it could possibly be an admin (or higher) command or require some sort of capability to keep people from being malicious.

**Comment by Mikaela** [comment]
I don't think this is useful as you can already configure how many mores you want.

```
< Mikaela> config help reply.mores.instant
< Limnoria> Determines how many mores will be sent instantly (i.e., without the use of the more command, immediately when they are formed). Defaults to 1, which means that a more command will be required for all but the first chunk.  (Current value: 1)
```

I personally have this as 2 as most of commands that are used on my bot usually only say link or something short after the `one more message`.

**Comment by monodemono** [comment]
Yeah, but sometimes you don't want all the mores at once, and sometimes you want more than you have it set to (2 in your case). Like, what if you have 5 pages of aliases, and you want to show them all to someone so they know the available commands? Instead of having to execute more 4 times, you can just do moreall. I admit these would be pretty fringe cases, but I don't think this command would go completely unused.
